index.html

```diff
@@ -263,5 +263,25 @@ <h1 class="first-title2">About Myself</h1>
         </div>
     </div>
 </section>
+<footer class="footer" id="contact">
+    <div class="container-3">
+        <div class="parent">
+            <img src="./Images/contactformshapes-mobile.png" class="over-img" alt="contactformimage"/>
```

Comment:
- [ Optional] This is a decorator image. Therefore, you should add it as a css background image to the footer instead.